NachoSpigot-Server/pom.xml

```diff
@@ -245,7 +243,7 @@
                                 </relocation>
                                 <relocation>
                                     <pattern>net.minecraft.server</pattern>
-                                    <shadedPattern>net.minecraft.server.v${minecraft_version}</shadedPattern>
+                                    <shadedPattern>net.minecraft.server.v1_8_R3</shadedPattern>
```

Comment:
On line 231
```patch
-                                    <shadedPattern>org.bukkit.craftbukkit.v${minecraft_version}</shadedPattern>
+                                    <shadedPattern>org.bukkit.craftbukkit.v1_8_R3</shadedPattern>
```

Reply:
what's this? i just removed the variable at the top and put its content where it was used, which is there

Reply:
I know, but you forgot to change it for craftbukkit's relocation.